src/condor_tools/htcondor_cli/annex.py

```diff
@@ -449,7 +463,11 @@ class add(Add):
         pass
 
 
+    class list(List):
+        pass
+
+
     @classmethod
     def verbs(cls):
-        return [cls.create, cls.add, cls.status, cls.shutdown]
+        return [cls.create, cls.add, cls.status, cls.shutdown, cls.list]
```

Comment:
Can you rename this to something other than list so you're not overriding a built-in type? `systems` might be good since it's providing a list of systems.